src/auto_renew_flows.ts

```diff
@@ -0,0 +1,139 @@
+import {
+  Block,
+  EventWithTransaction,
+  uint256,
+  formatUnits,
+} from "./common/deps.ts";
+import {
+  formatFelt,
+  SELECTOR_KEYS,
+  MONGO_CONNECTION_STRING,
+  AUTO_RENEW_CONTRACT,
+  DECIMALS,
+} from "./common/constants.ts";
+import { decodeDomain } from "./common/starknetid.ts";
+
+const filter = {
+  header: { weak: true },
+  events: [
+    {
+      fromAddress: formatFelt(AUTO_RENEW_CONTRACT),
+      keys: [formatFelt(SELECTOR_KEYS.ON_AUTO_RENEW_ENABLED)],
+    },
+    {
+      fromAddress: formatFelt(AUTO_RENEW_CONTRACT),
+      keys: [formatFelt(SELECTOR_KEYS.ON_AUTO_RENEW_DISABLED)],
+    },
+    {
+      fromAddress: formatFelt(AUTO_RENEW_CONTRACT),
+      keys: [formatFelt(SELECTOR_KEYS.ON_AUTO_RENEW_ENABLED)],
```

Comment:
there is twice the event `ON_AUTO_RENEW_ENABLED`, and the case `ON_AUTO_RENEW` is missing in the filters